priv/repo/migrations/20170630140136_create_project_github_repo.exs

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.Repo.Migrations.CreateProjectGithubRepo do
+  use Ecto.Migration
+
+  def change do
+    create table(:project_github_repos) do
+      add :project_id, references(:projects, on_delete: :delete_all)
+      add :github_repo_id, references(:github_repos, on_delete: :delete_all)
```

Comment:
`on_delete:` specifies what happens when the parent record, in this case, `GithubRepo` or `Project` gets deleted, not what happens when we delete this record.

We want to delete a `ProjectGithubRepo` whenever a `Project` or a `GithubRepo` get deleted.